### PR TITLE
Add Community-Supported Agriculture (CSA) category with 9 items

### PR DESCRIPTION
**Pull request by amo13**
Was looking for selfhostable server software for CSA purposes and found nothing on here. Some further research showed that quite a few interesting and alive projects exist. So I though I should add them to this awesome curated list for other people to find.

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
  - This is a new category with 9 initial items. Therefore keeping this grouped as one single PR.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of
  - [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management),
  - [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics),
  - [staticgen.com](https://www.staticgen.com/)
  - [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] The `Language` tag is the main server-side requirement for the software - don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by amo13** [comment]
Not sure why travis is failing. It says `The command "if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then make check_pr; fi" exited with 2.` but I don't know what it means.

File: README.md [hunk 1]
```diff
@@ -503,6 +504,20 @@ _[Extensible Messaging and Presence Protocol](https://en.wikipedia.org/wiki/XMPP
 - [Salut à Toi](http://www.salut-a-toi.org/) - Multipurpose, multi frontend, libre and decentralized communication tool. ([Source Code](http://repos.goffi.org/sat)) `AGPL-3.0` `Python`
   - [Libervia](http://wiki.goffi.org/wiki/Libervia/en) - Web frontend from Salut à Toi. ([Source Code](http://repos.goffi.org/libervia)) `AGPL-3.0` `Python`
 
+## Community-Supported Agriculture (CSA)
+
+**[`^        back to top        ^`](#)**
+
+- [ACP Admin](https://acp-admin.ch/) - CSA administration ([Source Code](https://github.com/acp-admin/acp-admin/)) `MIT` `Ruby`
```
Comment (n8225) [inline README.md:511]: Please elaborate the description and include a full stop ```.``` at the end of the description.

File: README.md [hunk 2]
```diff
@@ -503,6 +504,20 @@ _[Extensible Messaging and Presence Protocol](https://en.wikipedia.org/wiki/XMPP
...
+- [juntagrico](https://juntagrico.org/) - Management platform for community gardens and vegetable cooperatives. ([Source Code](https://github.com/juntagrico/juntagrico)) `LGPL-3.0` `Python`
+- [Local Food Nodes](https://localfoodnodes.org/) - Your open source platform for peoples driven local food markets and CSA. ([Source Code](https://gitlab.com/localfoodnodes/localfoodnodes)) `MIT` `PHP`
+- [Open Food Network](https://www.openfoodnetwork.org/) - Connect suppliers, distributors and consumers to trade local produce. ([Source Code](https://github.com/openfoodfoundation/openfoodnetwork)) `AGPL-3.0` `Ruby`
+- [OpenOlitor](https://openolitor.org/) - Administration platform for Community Supported Agriculture groups. ([Source Code](https://github.com/OpenOlitor)) `AGPL-3.0` `Scala`
+- [teikei](https://github.com/teikei/teikei) - A web application that maps out community-supported agriculture based on crowdsourced data. ([Demo](https://ernte-teilen.org/karte/#/)) `AGPL-3.0` `Javascript`
```
Comment (n8225) [inline README.md:519]: Language should be \`Nodejs\`

**Review by n8225 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -503,6 +504,22 @@ _[Extensible Messaging and Presence Protocol](https://en.wikipedia.org/wiki/XMPP
 - [Salut à Toi](http://www.salut-a-toi.org/) - Multipurpose, multi frontend, libre and decentralized communication tool. ([Source Code](http://repos.goffi.org/sat)) `AGPL-3.0` `Python`
   - [Libervia](http://wiki.goffi.org/wiki/Libervia/en) - Web frontend from Salut à Toi. ([Source Code](http://repos.goffi.org/libervia)) `AGPL-3.0` `Python`
 
+## Community-Supported Agriculture (CSA)
+
+**[`^        back to top        ^`](#)**
+
+_Management and administration tools for community supported agriculture and food cooperatives_
+
+- [ACP Admin](https://acp-admin.ch/) - CSA administration. ([Source Code](https://github.com/acp-admin/acp-admin/)) `MIT` `Ruby`
```
Comment (n8225) [inline README.md:513]: Could you elaborate a little more on the functuions of this application in the description.

**Review by n8225 — CHANGES_REQUESTED** [review]

**Review by n8225 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (amo13) [reply README.md]: Oh, I thought you meant a description for the whole category.
Item description now added too.

**Review by nodiscc — APPROVED** [review]
thanks

**Pull request merged**